setup.py

```diff
@@ -19,6 +19,7 @@
     "treeherder-client",
     "PGPy",
     "buildtools",
+    "python-jose",
```

Comment:
no, they are deps of python-jose so we don't need to be explicit here, but in puppet we don't auto-pull in deps on purpose